src/provider/default.ts

```diff
@@ -327,12 +340,10 @@ export class Provider implements ProviderInterface {
 
   public estimateFee(invocation: Invocation): Promise<any> {
     return this.fetchEndpoint('estimate_fee', undefined, {
-      // TODO: change the TYPE of the call
-      type: 'INVOKE_FUNCTION',
       contract_address: invocation.contractAddress,
-      entry_point_selector: getSelectorFromName(invocation.entrypoint),
-      calldata: bigNumberishArrayToDecimalStringArray(invocation.calldata ?? []),
-      signature: bigNumberishArrayToDecimalStringArray(invocation.signature ?? []),
+      entry_point_selector: invocation.entrypoint,
+      calldata: invocation.calldata ?? [],
+      signature: invocation.signature ?? [],
```

Comment:
why does this get removed?

Reply:
It was throwing 400 when testing. But it isn't now, so I brought it back